### PR TITLE
TPD-1393 Testing email parameter for method used by batch service

### PR DESCRIPTION
**Pull request by dahnbaez**
https://touchpointsoftware.atlassian.net/secure/RapidBoard.jspa?rapidView=5&projectKey=TPD&modal=detail&selectedIssue=TPD-1393&assignee=5e19301c85a8c90ecaca7d44
[touchpointsoftware.atlassian.net/browse/TPD-1393](https://touchpointsoftware.atlassian.net/browse/TPD-1393)

### CODE REVIEW

File: CmsData/Email/Emailer.cs [hunk 1]
```diff
@@ -579,13 +579,14 @@ public List<MailAddress> GetAddressList(Person p, string regemail = null)
         public void SendPeopleEmail(int queueid, bool onlyProspects = false)
         {
             var emailqueue = EmailQueues.Single(ee => ee.Id == queueid);
+            var isTestEmail = emailqueue.Testing ?? false;
             var from = Util.FirstAddress(emailqueue.FromAddr, emailqueue.FromName);
             if (!emailqueue.Subject.HasValue() || !emailqueue.Body.HasValue())
             {
                 SendEmail(from,
```
Comment (josearmandohf) [inline CmsData/Email/Emailer.cs:586]: Please Add a unit test for SendEmail()

File: CmsData/Email/Emailer.cs [hunk 2]
```diff
@@ -672,14 +673,14 @@ orderby t.PeopleId
...
-                            SendEmail(from, emailqueue.Subject, body, mal, emailqueue.Id, cc: cc);
+                            SendEmail(from, emailqueue.Subject, body, mal, emailqueue.Id, cc: cc, testEmail: isTestEmail);
                         }
                     }
                     catch (Exception ex)
                     {
                         var subject = $"sent emails - error:(emailid={emailqueue.Id}) {CmsHost}";
                         ErrorLog.GetDefault(null).Log(new Error(new Exception(subject, ex)));
-                        SendEmail(from, subject, ex.Message, Util.ToMailAddressList(from));
+                        SendEmail(from, subject, ex.Message, Util.ToMailAddressList(from), testEmail: isTestEmail);
```
Comment (josearmandohf) [inline CmsData/Email/Emailer.cs:683]: Same here

**Review by josearmandohf — CHANGES_REQUESTED** [review]
Can you add the Unit/Integration tests required?

**Review by josearmandohf — CHANGES_REQUESTED** [review]
Please add a unit test for...

SendEmail(from, subject, ex.Message, Util.ToMailAddressList(from), testEmail: isTestEmail);

**Review by brondavies — APPROVED** [review]

**Pull request merged**